### DISCUSSION TITLE
[Feature] Sidebar: Add Videos as Entry

### DISCUSSION THREAD

**Discussion by MickLesk** [immich-app/immich#8745] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello dear development team,

How about an additional tab "Videos" in the sidebar? 
All video files (or live photos?) are displayed there.

In connection with this feature request: 
https://github.com/immich-app/immich/discussions/8743
it would also be worth considering implementing a submenu where the albums in which the videos are located are displayed (maximum 5 or "all albums").

If you click on "all albums", it would even be cool to display this here: 
https://github.com/immich-app/immich/discussions/5409

<img width="127" alt="image" src="https://github.com/immich-app/immich/assets/47820557/d32733c8-fa56-460c-8711-ad2dee93df68">

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by MickLesk** [comment]
Solved with the PR:
https://github.com/immich-app/immich/pull/13981

**Reply by megaeverything** [reply]
Was this feature ever added? I cant find a way to have Videos show up in the sidebar like the screenshot in your request.

**Reply by MickLesk** [reply]
Nope
